src/pibasis.jl

```diff
@@ -166,6 +166,19 @@ setreal(basis::PIBasis, isreal::Bool) =
 
 maxcorrorder(basis::PIBasis) = maxcorrorder(basis.spec)
 
+getval(A2Bmap) =  [norm(A2Bmap[i,j].val) for i = 1:size(A2Bmap)[1], j = 1:size(A2Bmap)[2]]
```

Comment:
(1) there is no connection here to PIBasis, hence it shouldn't be in this file, it should be in `symmbasis.jl`
(2)   the name is far too generic. Please give it a descriptive name and start with an underscore, e.g. `_scaling_get_a2b_vals` or something like that which makes it effectively private.

Reply:
Will do soon, thanks for the suggestion.